Add Actions on Google

ravirupareliya opened the pull request:
https://github.com/ravirupareliya/awesome-aog

A curated list of awesome AoG(Action on Google) libraries, blogs, open-source project, videos.

PRs reviewed:
[#1613 (comment)](https://github.com/sindresorhus/awesome/pull/1613#issuecomment-530035621)
[#1606 (comment)](https://github.com/sindresorhus/awesome/pull/1606#issuecomment-529618538)
[#1612 (comment)](https://github.com/sindresorhus/awesome/pull/1612#issuecomment-529615844)

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.


## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- **If you use a license badge, it should be SVG, not PNG.**
	- To verify that you've read all the guidelines, please add comment with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

ravirupareliya commented:
unicorn

ravirupareliya commented:
@sindresorhus Thank you for spending time on reviewing it.

> I think the repo name should be awesome-actions-on-google. aog is meaningless.

Kept it as AoG because all the Actions on Google developers including googlers are referring it as AoG only. As per your suggestion I have changed it to the awesome-actions-on-google

ravirupareliya commented:
@sindresorhus Anything else that you believe needs to be updated or corrected for this entry?

sindresorhus commented:
> > Includes a succinct description of the project/theme at the top of the readme. (Example)

> Not done.

Still not done.

---

The entries in https://github.com/ravirupareliya/awesome-actions-on-google#open-source-code needs a description.

ravirupareliya commented:
> > > Includes a succinct description of the project/theme at the top of the readme. (Example)
> 
> > Not done.
> 
> Still not done.

It is there in repo. Text is "A curated list of awesome AoG(Action on Google) libraries, blogs, open-source project, videos."
> 

> The entries in https://github.com/ravirupareliya/awesome-actions-on-google#open-source-code needs a description.
It is examples provided by google, do you want me to add short descriptions of those projects?

sindresorhus commented:
> It is there in repo. Text is "A curated list of awesome AoG(Action on Google) libraries, blogs, open-source project, videos."

Yes, but it's not correct. It should describe what "Action on Google" is, not what your list contains. This clearly shown even with illustrated examples in the above guidelines.

> It is examples provided by google, do you want me to add short descriptions of those projects?

Yes

ravirupareliya commented:
@sindresorhus 
> > It is there in repo. Text is "A curated list of awesome AoG(Action on Google) libraries, blogs, open-source project, videos."
> 
> Yes, but it's not correct. It should describe what "Action on Google" is, not what your list contains. This clearly shown even with illustrated examples in the above guidelines.
> 
Got it, updated it now. Sorry I was thinking it should describe what my list is containing.
> > It is examples provided by google, do you want me to add short descriptions of those projects?
> 
> Yes
Updated

ravirupareliya commented:
I have linting error in the heading section because it has ?(question mark), but I have seen it in lot many other entries.
example : https://github.com/mailtoharshit/awesome-salesforce

bsoyka commented:
I'm only seeing one linter error. On the ninth item of [this section](https://github.com/ravirupareliya/awesome-actions-on-google/blob/7a599a211576670bfb0d107da5018168c7227e94/README.md#open-source-code), "Youtube" needs to be capitalized as "YouTube".
___
Other than that, I think this looks like a good start.

ravirupareliya commented:
> I'm only seeing one linter error. On the ninth item of [this section](https://github.com/ravirupareliya/awesome-actions-on-google/blob/7a599a211576670bfb0d107da5018168c7227e94/README.md#open-source-code), "Youtube" needs to be capitalized as "YouTube".
> 
Updated.

@sindresorhus

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1177088847601004545

ravirupareliya commented:
Thank you for accepting the request.
